olp-cpp-sdk-core/tests/logging/MessageFormatterTest.cpp

```diff
@@ -0,0 +1,501 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <gtest/gtest.h>
+#include <olp/core/logging/MessageFormatter.h>
+
+using namespace ::testing;
+
+
+namespace olp {
+namespace logging {
+
+TEST( MessageFormatterTest, ElementConstructors )
```

Comment:
```suggestion
TEST(MessageFormatterTest, ElementConstructors)
```
Please, run clang-format on this file.